mmv1/products/datastream/PrivateConnection.yaml

```diff
@@ -43,6 +43,7 @@ custom_code: !ruby/object:Provider::Terraform::CustomCode
   constants: templates/terraform/constants/private_connection.go.erb
   post_create: templates/terraform/post_create/private_connection.go.erb
   post_import: templates/terraform/post_import/private_connection.go.erb
+  pre_delete: templates/terraform/pre_delete/private_connection.go.erb
```

Comment:
you could achieve the same through a  delete url. It might be more readable this way !!

https://github.com/GoogleCloudPlatform/magic-modules/blob/5043710da3cb65c093a0dd527c7d857b9bc6d697/mmv1/products/bigquery/go_Dataset.yaml#L32

:)